From 2c4645a377d179dbb11b65ab462c1d514554c0e0 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:44 -0400
Subject: [redhat] kabi: Adding symbol put_page (mm/swap.c)

Message-id: <131d951443b0e5a81e4002c9cd0aace667f0af0e.1410510773.git.poros@redhat.com>
Patchwork-id: 96072
O-Subject: [PATCH RHEL7.1 164/233] kabi: Adding symbol put_page (mm/swap.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    To release the page received through get_user_page.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/put_page
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/put_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/put_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/put_page  b/redhat/kabi/kabi-module/kabi_ppc64/put_page 
new file mode 100644
index 0000000..d67b218
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/put_page  
@@ -0,0 +1,2 @@
+#1-
+0xf72dac45 put_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/put_page  b/redhat/kabi/kabi-module/kabi_s390x/put_page 
new file mode 100644
index 0000000..be5d1a3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/put_page  
@@ -0,0 +1,2 @@
+#1-
+0x5da0bf48 put_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/put_page  b/redhat/kabi/kabi-module/kabi_x86_64/put_page 
new file mode 100644
index 0000000..651ee52
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/put_page  
@@ -0,0 +1,2 @@
+#1-
+0x334c1f75 put_page vmlinux EXPORT_SYMBOL
-- 
1.7.1